docs/Writing_yeb_easyconfig_files.rst

```diff
@@ -261,24 +284,72 @@ A more complicated example from a toolchain easyconfig, where also the ``!join``
         - &comp_version 4.7.2
         - &comp [*comp_name, *comp_version]
 
-        - &blaslib OpenBLAS
+        - & OpenBLAS
         - &blasver 0.2.6
         - &blas !join [*blaslib, -, *blasver]
         - &blas_suff -LAPACK-3.4.2
 
         - &comp_mpi_tc [gompi, 1.4.10]
 
-    dependencies: [
-        *comp,
-        [OpenMPI, 1.6.4, '', *comp],
-        [*blaslib, *blasver, *blas_suff, *comp_mpi_tc],
-        [FFTW, 3.3.3, '', *comp_mpi_tc],
-        [ScaLAPACK, 2.0.2, !join [-, *blas, *blas_suff], *comp_mpi_tc]
-    ]
+    dependencies:
+        - *comp_name: *comp_version
+        - OpenMPI: 1.6.4
+          toolchain: *comp
+        - *blaslib: *blasver
+          versionsuffix: *blas_suff
+          toolchain: *comp_mpi_tc
+        - FFTW: 3.3.3
+          toolchain: *comp_mpi_tc
+        - ScaLAPACK: 2.0.2
+          versionsuffix: !join [-, *blas, *blas_suff]
+          toolchain: *comp_mpi_tc
 
 For the full version of this easyconfig file, see the example ``.yeb`` easyconfig
 :ref:`easyconfig_yeb_format_examples_goolf1410`.
 
+.. _easyconfig_yeb_format_new:
+
+OS dependencies and sanity check paths
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+To specify parameters that used to contain tuples such as ``osdependencies`` and ``sanity_check_paths``, simply use
+list (sequences) instead of tuples.
+
+For example::
+
+    osdependencies = [('openssl-devel', 'libssl-dev', 'libopenssl-devel')]
+
+Becomes::
+
+    osdependencies: [[openssl-devel, libssl-dev, libopenssl-devel]] 
+
+And::
+
+    sanity_check_paths = {
+        'files': ['fileA', ('fileB', 'fileC')],
+        'dirs' : ['dirA', 'dirB'],
+    }
+
+Becomes::
+
+    sanity_check_paths: {
+        files: [fileA, [fileB, fileC]],
+        dirs: [dirA, dirB]
+    }
+
+Shorthands for common easyconfig parameters
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Toolchain format
+################
+
+The easyconfig parameter ``toolchain`` in .eb files is defined as a dictionary ``{'name':'foo', 'version':'bar'}``. In
+the .yeb format, this can be done much easier by just using ``name, version``. E.g::
+
+    toolchain = {'name':'intel', 'version':'2015b'}
```

Comment:
same here, clearly mention this is `.eb` syntax rather than `.yeb`, otherwise this is going to be very confusing (people tend to only partially read documentation and look at examples instead)

So, maybe use this:

```
# note: this is .eb format, this will *not* work in .yeb files
toolchain = {'name':'intel', 'version':'2015b'}
```